README.md

```diff
@@ -65,6 +67,16 @@ def deps do
 end
 ```
 
+## Building on Windows
+
+Building on windows requires the Microsoft build tools (for C++/C) and `mix deps.compile` being run in a shell where the environment is set up correctly. This can be done by running the `vcvarsall.bat` script provided by the framework. The quickest way to get that is to paste this (replace the visual studio install path accordingly) into the `run` dialog (Win+R).
+
+```bat
+cmd /K "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" amd64
+```
+
+Currently `mix compile` only succeeds on the 2nd run. That means `mix deps.compile` has to be run twice.
```

Comment:
I'd like to try and sort this out - what happens on the first run? Does it actually build, or does it fail the first time but succeed the second? Perhaps its related to the conditional code in the `Makefile.win` that builds differently when `Makefile.win.auto` is present? I'm happy to try and help troubleshoot, I just don't have a Windows VM handy at the moment to dive in myself.

Reply:
I think the solution is hinted at by [these lines in the docs](https://github.com/elixir-lang/elixir_make/blob/f2f94d68c0a3ef494c838535139a5e2256656dd8/lib/mix/tasks/compile.make.ex#L88-L98). We can remove the `priv` directory and instead explicitly copy artifacts to `$MIX_APP_PATH/priv` in the targets.

Reply:
> I'd like to try and sort this out - what happens on the first run?

It builds everything, also generates dll, exp and lib files in the priv directory. It just doesn't copy them, maybe it's because of the warnings

It still happens after switching the makefile to use the environment variables from mix...

Reply:
> I think the solution is hinted at by [these lines in the docs](https://github.com/elixir-lang/elixir_make/blob/f2f94d68c0a3ef494c838535139a5e2256656dd8/lib/mix/tasks/compile.make.ex#L88-L98). We can remove the `priv` directory and instead explicitly copy artifacts to `$MIX_APP_PATH/priv` in the targets.

Finally success, this worked.

the `ERTS_INCLUDE_DIR` variable contains linux directory node separators (forward slashes) and while `cl` seems to understand those, `copy` does not. So I made `cl` output directly into the `_build` directory as defined by the `MIX_APP_PATH` variable

This works now (on my system...):
```cmd
del /F /Q /S _build
mix deps.compile
mix compile
mix test
echo %errorlevel%
rem (prints `0`)
```

So it's just a single compile now

I'll also create a ticket on elixir_make - `ERTS_INCLUDE_DIR` (and others) should really contain Windows path elements on Windows...